cpp/src/worker_progress_thread.cpp

```diff
@@ -49,6 +53,7 @@ WorkerProgressThread::~WorkerProgressThread()
   }
 
   _stop = true;
+  _signalWorkerFunction();
   _thread.join();
 }
```

Comment:
Should the `_thread`'s destructor be called when deleting the worker progress thread?

Reply:
Not sure I understand the question. The `_thread` destructor will run when the `~WorkerProgressThread` goes out-of-scope, did you expect something different?